common/config/environment.go

```diff
@@ -105,10 +105,11 @@ func (e *CommonEnvironment) ResourcesTags() pulumi.StringMap {
 	defaultTags["username"] = pulumi.String(user.Username)
 
 	// Map environment variables
-	lookupVars := []string{"DD_TEAM"}
+	lookupVars := []string{"TEAM", "PIPELINE_ID"}
```

Comment:
This changes from `DD_TEAM` to `TEAM`, is that intentional?

Reply:
Yes. `DD_` prefix seemed unnecessary. I checked and could not find usage of this tag anywhere. So I changed it to `TEAM`. 
No strong reason for changing it however.